python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -37,9 +37,9 @@ def __init__(
         level : int, level_name or list, optional
             For objects with a MultiIndex, `level` can be used to specify
             grouping by one or more levels of the MultiIndex.
-        sort : True, optional
-            If True (default), sort results by group9s). Note that
-            unlike Pandas, this also sorts values within each group.
```

Comment:
What does `also sorts values within each group` mean?

Reply:
I'm not sure here... @shwina @brandon-b-miller @thomcom does this ring a bell for any of you?

Reply:
No idea... I think we can safely get rid of this as it sounds like an additional guarantee.